examples/nextjs-blog/pages/blog/list.js

```diff
@@ -1,20 +1,58 @@
+import { flamelinkApp as app } from '../../utils/flamelink'
 import { Link } from '../../routes'
+import { getPostsWithMedia, getImageAlt } from '../../utils/post/post.util'
 
 const Posts = function(props) {
-  return Object.values(props.posts).map(post => (
-    <h3>
-      <Link route={`/blog/${post.slug}`}>{post.title}</Link>
-    </h3>
-  ))
+  const { posts } = props
+
+  if (posts && posts.length > 0) {
+    return Object.values(props.posts).map(post => {
+      const {
+        title,
+        slug,
+        author: { displayName },
+        excerpt,
+        _fl_meta_: { docId },
+        imageURL,
+        date,
+      } = post
+
+      return (
+        <div key={docId}>
+          <Link route={`/blog/${slug}`}>{`${title} - ${displayName}`}</Link>
+          <p>{excerpt}</p>
+          <img src={imageURL} alt={getImageAlt(post)} />
+          <p>{new Date(date).toLocaleDateString()}</p>
+        </div>
+      )
+    })
+  }
+
+  return 'No blog posts!'
 }
 
-Posts.getInitialProps = async ({ query }) => {
+Posts.getInitialProps = async () => {
+  const posts = await app.content.get({ schemaKey: 'blogPost', populate: true })
```

Comment:
can we add the sdk filter for this and not filter client side as an example?

Reply:
@jperasmus how do we do a where lookup and orderBy for CF?

Might be something to add to the docs

`
app.content.get({ ..., where: ["status == 'published'"], orderBy: [{ field: 'date',  order: 'desc'}] })
`

Reply:
ah ofc, dankie

Reply:
we have the filters
```
app.content.get({ 
  ..., 
  filters: [ ["status", "==" "published"] ], 
  orderBy: [{ field: 'date', order: 'desc'}] 
})
```

Reply:
yup, lekker, you've got it @gitdubz